lightning/src/ln/channel.rs

```diff
@@ -1006,7 +1014,7 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 			lock_time: ((0x20 as u32) << 8*3) | ((obscured_commitment_transaction_number & 0xffffffu64) as u32),
 			input: txins,
 			output: outputs,
-		}, non_dust_htlc_count, htlcs_included)
+		}, non_dust_htlc_count, htlcs_included, scripts)
```

Comment:
Does that affect the signer API?

Reply:
In-flight dust HTLCs are added in fees, so policy signer may want to authorize a higher fee to take them in account. Though that opens vector attack so some signers may want to forbid dusts. You may also check fees range with fees=fees+max_dust and so check against this supplemental provided value.

Reply:
It seems that the existing arguments to `sign_remote_commitment` already allow this kind of policy check, since the signer can just look at the transaction outputs check which of them dust.  Am I missing something?

Reply:
You're right! htlcs_included += included_dust_htlcs in build_commitment_transaction, didn't think so from memory